ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -55,6 +55,7 @@ statement
     | (LIST | SHOW) TYPES                                                   #listTypes
     | (LIST | SHOW) VARIABLES                                               #listVariables
     | DESCRIBE EXTENDED? sourceName                                         #showColumns
+    | DESCRIBE EXTENDED? STREAMS                                            #describeStreams
```

Comment:
Won't this syntax going to confuse users between the use of `show streams extended` and `describe extended streams`? fyi @rodesai

I was getting confused too, so I wonder how users would react to this.

Reply:
Yeah, I do think it's somewhat confusing. It's sticking with the current semantic set up though since we have `describe extended sourcename` verses `list streams extended` so I'm not sure it would make sense to switch it to `describe streams extended` if we don't also switch it to `describe sourceName extended`. We briefly mentioned this when talking to @derekjn and @colinhicks but there wasn't a full discussion